.travis.yml

```diff
@@ -1,5 +1,12 @@
 language: python
-sudo: false
+
+matrix:
+  include:
+  - sudo: required
+    services:
+      - docker
+    env: PIP=pip
```

Comment:
The `matrix` block is not needed. `$PIP` usage below should be replaced with `python -m pip install cibuildwheel twine` to prevent Travis failure.